runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/config/GuardConfig.java

```diff
@@ -17,13 +17,20 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.net.URL;
+import java.util.function.Function;
+
+import jakarta.json.bind.Jsonb;
+
 public class GuardConfig
 {
     public transient long id;
 
     public final String name;
     public final String type;
     public final OptionsConfig options;
+    public Function<URL, String> readURL;
+    public Jsonb jsonb;
```

Comment:
Please mark these as `transient` and move up alongside other `transient` fields.